src/handlers/handle-scheduled-checkin.ts

```diff
@@ -92,6 +92,8 @@ async function handleInternal(event: AWSLambda.SQSEvent) {
     Luxon.DateTime.now().toLocaleString(Luxon.DateTime.DATETIME_FULL_WITH_SECONDS)
   );
 
+  // Perform the checkin. Since we've already gotten a response indicating that the checkin is
+  // ready, we only need to make one request.
```

Comment:
Nice comments, I've always mentally visualized that the above step is actually doing the check-in, I didn't realize it's just checking if it's ready and then we do the check-in after the fact.